ssv/committee_runner.go

```diff
@@ -212,22 +216,26 @@ func (cr CommitteeRunner) ProcessPostConsensus(signedMsg *types.PartialSignature
 			if cr.HasSubmitted(role, validator, slot) {
 				continue
 			}
+
+			// Reconstruct signature
 			share := cr.BaseRunner.Share[validator]
 			pubKey := share.ValidatorPubKey
 			sig, err := cr.BaseRunner.State.ReconstructBeaconSig(cr.BaseRunner.State.PostConsensusContainer, root,
 				pubKey[:], validator)
-			// If the reconstructed signature verification failed, fall back to verifying each partial signature
-			// TODO should we return an error here? maybe other sigs are fine?
 			if err != nil {
+				// If fail, fall back to verifying each partial signature
 				for root := range rootSet {
 					cr.BaseRunner.FallBackAndVerifyEachSignature(cr.BaseRunner.State.PostConsensusContainer, root,
 						share.Committee, validator)
 				}
-				return errors.Wrap(err, "got post-consensus quorum but it has invalid signatures")
+				// Record the error and continue to next validators
+				anyErr = errors.Wrap(err, "got post-consensus quorum but it has invalid signatures")
```

Comment:
i think `errors.Join` is preferable as these are multiple errors rather than nested errors

Reply:
@MatheusFranco99 we said we need to update go version for this correct?